### PR TITLE
Bootstrap modal-backdrop doesn't disappear after creating a new aspect #6562

### PR DESCRIPTION
**Pull request by deXterbed**

### CODE REVIEW

File: lib/assets/javascripts/jquery.mentionsInput.js [hunk 1]
```diff
@@ -95,6 +97,7 @@
       elmInputBox.attr('data-mentions-input', 'true');
       elmInputBox.bind('keydown', onInputBoxKeyDown);
       elmInputBox.bind('keypress', onInputBoxKeyPress);
+      elmInputBox.bind('paste',onInputBoxPaste);
```
Comment (diaspora-code-review) [inline lib/assets/javascripts/jquery.mentionsInput.js]: Strings must use doublequote.

File: app/assets/templates/aspect_create_modal_tpl.jst.hbs [hunk 2]
```diff
@@ -42,7 +42,7 @@
         <div class="btn btn-default" data-dismiss="modal" aria-hidden="true">
           {{ t "cancel" }}
         </div>
-        <div class="btn btn-primary">
+        <div class="btn btn-primary" data-dismiss="modal" data-backdrop="false">
```
Comment (svbergerem) [inline app/assets/templates/aspect_create_modal_tpl.jst.hbs]: AFAIK `data-backdrop="false"` doesn't do anything here. You can add it to buttons which open the modal to disable the backdrop.

**Comment by denschub** [comment]
@deXterbed Could you please address the comment?

**Comment by svbergerem** [comment]
@denschub Before moving on with this PR I'd like to see an answer to [my comment in #6562](https://github.com/diaspora/diaspora/issues/6562#issuecomment-166131159).

**Comment by denschub** [comment]
Okey-doke!

**Comment by deXterbed** [comment]
@denschub i had my notifications turned off for some reason, replied to @svbergerem's comment just now

**Comment by svbergerem** [comment]
Superseded by #6864.

**Pull request closed without merging**